core/services/offchainreporting/run_saver.go

```diff
@@ -32,12 +33,14 @@ func (r *RunResultSaver) Start() error {
 			for {
 				select {
 				case run := <-r.runResults:
-					r.logger.Infow("RunSaver: saving job run", "run", run)
-					// We do not want save successful TaskRuns as OCR runs very frequently so a lot of records
-					// are produced and the successful TaskRuns do not provide value.
-					if err := r.pipelineRunner.InsertFinishedRun(&run, false); err != nil {
-						r.logger.Errorw("error inserting finished results", "err", err)
-					}
+					recovery.WrapRecover(r.logger, func() {
```

Comment:
This is so simple and doesnt process any outside input I think its reasonable to not wrap here

Reply:
Done